### ISSUE TITLE
Add a new source field to the store updates

### ISSUE THREAD

**Issue by arjun27** [covid-maps/covid-maps#243]
...and show it on the UI. For data points that we get from other sources.

**Comment by asanjeevrao** [comment]
https://knowmea.com/home/covid19/ - someone we want to collab with didn't mention it explicitly but I think is expecting a link to be shared. Is that possible? Have an icon like this next to the text that can take users to the URL of the source 
![image](https://user-images.githubusercontent.com/39347351/79396537-014a6800-7f31-11ea-9c1d-da00e8c5f563.png)

**Comment by sampoorna** [comment]
Yeah that's totally possible. If the source is a link. But it may not always be.
